src/formats/__tests__/json-schema.spec.ts

```diff
@@ -0,0 +1,197 @@
+import {
+  isJSONSchema,
+  isJSONSchemaDraft4,
+  isJSONSchemaDraft6,
+  isJSONSchemaDraft7,
+  isJSONSchemaLoose,
```

Comment:
Can we add support for 2019-09? It came out yesterday.

Reply:
Sure, what's the URL?
` 'http://json-schema.org/draft-2019-09/schema#'`?

Reply:
http://json-schema.org/